aws_lambda_powertools/utilities/data_classes/code_deploy_lifecycle_hook_event.py

```diff
@@ -0,0 +1,13 @@
+from aws_lambda_powertools.utilities.data_classes.common import DictWrapper
+
+
+class CodeDeployLifeCycleHookLambdaEvent(DictWrapper):
```

Comment:
```suggestion
class CodeDeployLifecycleHookEvent(DictWrapper):
```

What do you think about this name?